addons/website/views/website_views.xml

```diff
@@ -619,9 +619,15 @@
             name="Edit Menu"
             action="website.website_editor"/>
 
-        <menuitem id="menu_page_properties"
```

Comment:
Maybe we should avoid a diff on this line by adding "menu_ace_editor" after "menu_page_properties" instead of removing it to put it back later (to see which order is best for UX)